lib/kubecfg.libsonnet

```diff
@@ -144,6 +144,37 @@
       error ('o must be an object or array of k8s objects, found ' + std.type(o))
   ),
 
+  local objectGetDeep(o, f, default) = (
```

Comment:
I gave it some thought and I think for consistency with the jsonnet stdlib we should do this:

- `kubecfg.getPath(o, p, default=null, inc_hidden=true)` would behave like `std.get` but with a path
- `kubecfg.objectHasPath(o, p, inc_hidden=true)` like `std.objectHas` but allowing the user to ignore hidden fields like in `std.get` `inc_hidden`
- `kubecfg.objectHasPathAll(o, p)` as a simple wrapper for `kubecfg.objectHasPath(o,p,inc_hidden=false)`

Reply:
Whilst I was having lunch I thought of the first one as well :laughing: 

I'll tweak this a little bit and get it to fit more inline with the `std` implementations :+1: